install/kubernetes/helm/skywalking/templates/NOTES.txt

```diff
@@ -15,22 +15,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */}}
 
-Get the UI URL by running these commands:
+
+Thank you for installing {{ .Chart.Name }}.
+
+Your release is named {{ .Release.Name }}.
+
+Learn more, please visit https://skywalking.apache.org/
+
 {{- if .Values.ui.ingress.enabled }}
+Get the UI URL by running these commands:
 {{- range .Values.ui.ingress.hosts }}
   http{{ if $.Values.ui.ingress.tls }}s{{ end }}://{{ . }}{{ $.Values.ui.ingress.path }}
 {{- end }}
-{{- else if contains "NodePort" .Values.ui.service.type }}
```

Comment:
Why remove them?

Reply:
I feel that it can't give the user a good hint.

Reply:
I think https://github.com/apache/skywalking/blob/istio-ci/install/kubernetes/helm/skywalking/templates/NOTES.txt will give the user an instrument on how to access the cluster. 
BTW, our several international users are picking up `LoadBalancer` as the UI endpoint. So we can't remove it.